numpy/random/mtrand/mtrand.pyx

```diff
@@ -1159,22 +1159,29 @@ cdef class RandomState:
         """
         return disc0_array(self.internal_state, rk_long, size, self.lock)
 
-    def randint(self, low, high=None, size=None, dtype='l'):
+    def randint(self, low=None, high=None, size=None, dtype='l'):
         """
-        randint(low, high=None, size=None, dtype='l')
+        randint(low=None, high=None, size=None, dtype='l')
 
         Return random integers from `low` (inclusive) to `high` (exclusive).
 
         Return random integers from the "discrete uniform" distribution of
-        the specified dtype in the "half-open" interval [`low`, `high`). If
-        `high` is None (the default), then results are from [0, `low`).
+        the specified dtype in the "half-open" interval [`low`, `high`).
+
+        If ``low`` and ``high`` are None (default), the results are from
+        [`lowbnd`, `highbnd`), where `lowbnd` and `highbnd` are equal to
+        `np.iinfo(dtype).min` and `np.iinfo(dtype).max`.
+
+        If only ``low`` is None, the results are from [`lowbnd`, `high`),
+        where `lowbnd` is defined as previously. If only `high` is None,
+        then if `low` > 0, results are from [0, `low`). Otherwise, results
+        are from [`low`, `highbnd`), where `highbnd` is defined as previously.
 
         Parameters
         ----------
-        low : int
-            Lowest (signed) integer to be drawn from the distribution (unless
-            ``high=None``, in which case this parameter is the *highest* such
-            integer).
+        low : int, optional
+            If provided, the lowest (signed) integer to be drawn from the
```

Comment:
Yeah, probably should.  I don't see it having too much of any impact on the meaning.